src/modules/bill-runs-import/lib/queries.js

```diff
@@ -150,7 +158,15 @@ nbt2.charge_type,
 nbt2.net_amount,
 nbt2.legacy_id,
 nbt2.description,
-row_to_json(nbt) as metadata
+row_to_json(nbt) as metadata,
+nbt2.source_value,
+nbt."SEAS_VALUE"::numeric,
+nbt."LOSS_VALUE"::numeric,
+nbt2.suc_rate,
+case when nbt."ELEMENT_AGRMNTS" LIKE '%126' THEN REPLACE(nbt."ELEMENT_AGRMNT_VALS", 'x ', '')::numeric ELSE null END,
+case when nbt."ELEMENT_AGRMNTS" LIKE '%127' THEN REPLACE(nbt."ELEMENT_AGRMNT_VALS", 'x ', '')::numeric ELSE null END,
```

Comment:
I think the `S126` factor is probably OK here as RK mentioned it will apply to standard and compensation charges.
I think the `S127` one might need to only apply to standard charges - and therefore may need to move to the block below.

Essentially you get:
Annual:
50% standard charge
100% compensation charge

TPT:
50 % standard charge x actual usage

Reply:
Hello!
I've modified the S127 bit
As for S130: We can bring this in if needed - but we haven't done that for the CM bit either so I would suggest it should be its own mini ticket so we can create the relevant columns in the water schema :slightly_smiling_face:
Part of me assumed it will happen as part of the abatements work which is coming at some point in the future